### PR TITLE
remove gulp-clean-css. add gulp-cssnano.

### PR DESCRIPTION
**Pull request by scniro**

### CODE REVIEW

File: src/blackList.json [hunk 1]
```diff
@@ -278,6 +278,6 @@
   "gulp-uglifyjs-wrapper": "duplicate of gulp-uglify",
   "gulp-rollup": "use the `rollup` module directly",
   "gulp-livereload": "abandoned. use gulp-refresh",
-  "gulp-minify-css": "deprecated. use gulp-cssnano.",
```
Comment (phated) [inline src/blackList.json]: you accidentally removed `gulp-minify-css` and not `gulp-clean-css`

Comment (scniro) [inline src/blackList.json]: @phated `gulp-minify-css` is the old, `gulp-clean-css` is the new, the package that I am now maintaining, so this is correct

**Comment by scniro** [comment]
@phated Ah, you're right I totally spaced on that one. Should be okay now

File: src/blackList.json (hunk 1 above)
Comment (scniro) [inline src/blackList.json]: @phated my bad on that - totally spaced for a moment, or two

**Comment by phated** [comment]
@scniro please fix the diff, there's no need to move gulp-minify-css to the bottom.  Also, update its description to recommend gulp-clean-css

**Comment by scniro** [comment]
@phated done. All should be well - I can only hope at this point T_T

File: src/blackList.json [hunk 2]
```diff
@@ -269,7 +268,7 @@
   "gulp-tsb": "duplicate of gulp-typescript",
   "gulp-tsc-sushicutta": "duplicate of gulp-typescript",
   "gulp-type": "renamed to gulp-typescript",
-  "gulp-typescript-alpha-1.5.0": "duplicate of gulp-typescript",
+  "gulp-typescript-alpha-1.5.0": "duplicate of gulp-typescricpt",
```
Comment (phated) [inline src/blackList.json]: somehow ended up adding a typo here

**Comment by phated** [comment]
1 problem was introduced (commented inline).  Can you squash these commits into 1?  Thanks.

**Pull request closed without merging**